python/cudf_polars/tests/expressions/test_agg.py

```diff
@@ -96,7 +96,7 @@ def test_bool_agg(agg, request):
     assert_gpu_result_equal(q, check_exact=False)
 
 
-@pytest.mark.parametrize("cum_agg", expr.UnaryFunction._supported_cum_aggs)
+@pytest.mark.parametrize("cum_agg", sorted(expr.UnaryFunction._supported_cum_aggs))
```

Comment:
Thanks, good spot!

Reply:
For reference, this was necessary to make multi-worker testing operate correctly. The `frozenset` had a different order on each worker, so pytest failed because the test collection didn't agree across all workers.